lib/api_stream_test.go

```diff
@@ -0,0 +1,142 @@
+package sebak
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"boscoin.io/sebak/lib/block"
+	observable "github.com/GianlucaGuarini/go-observable"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPIStreamRun(t *testing.T) {
+	tests := []struct {
+		name       string
+		makeStream func(*observable.Observable) *EventStream
+		trigger    func(*observable.Observable)
+		respFunc   func(testing.TB, *http.Response)
+	}{
+		{
```

Comment:
While the double block ? It's rather hard to read

Reply:
IMHO, To write is harder to read.. hahaha ㅠㅠ